apps/sms/views/conversation/js/conversation.js

```diff
@@ -622,29 +625,31 @@ var ConversationView = {
     // to slide correctly. Bug 1009541
     this.cancelEdit();
 
-    if (Navigation.isCurrentPanel('thread')) {
```

Comment:
and actually, we need this condition... but different ! we should not revoke the URL if we go to group or report view... try this STR: have a MMS with an image, that is not displayed when loading the conversation (because it's too high in the conversation); then display the report for a displayed message. Come back to the conversation, and scroll up to show the MMS => boom, there is no image displayed ! I'll file a separate bug and this should be quite easy to fix :)